File: Armstrong_number/armstrong_number.js
let number = parseINT(prompt("Enter a number: "));
let temp = number
let armstrong_number = 0;

while(temp != 0){
    let r = temp % 10;
    armstrong_number += r ** 3;
    temp = Math.floor(temp / 10);
}

console.log(armstrong_number);

File: Armstrong_number/Armstrong_number.c
#include<stdio.h>

int main(){
    int number, r, temp, sum = 0;
    scanf("%d", &number);

    temp = number;

    while(temp != 0){
        r = temp % 10;
        sum = sum + r*r*r;
        temp = temp / 10;
    }
    printf("%d", sum);

    return 0;
}